### DISCUSSION TITLE
LXC Build Script

### DISCUSSION THREAD

**Discussion by DocDrydenn** [rommapp/romm#344] [Ideas]
Any chance of an official non-Docker build or set of instructions? I've been migrating away from Docker and pushing more into the Proxmox LXC world. Would love to get this moved into a LXC as well.

Bonus: This would allow #tteck to create a RomM script to add to his awesome Proxmox scripts repo. 
- [tteck/Proxmox](https://github.com/tteck/Proxmox) 
- [http://helper-scripts.com/](http://helper-scripts.com/)

**Comment by zurdi15** [comment]
Omg I am sorry, saw this when you opened the discussion but totally forgot about it, I apologize. 

We don't have plans for that in the short term as any of the development team is currently using it, so it's hard to test (already supporting unraid with the same problem and struggling a lot supporting the unraid community).

But eventually we want to reach as much systems as possible, so I would say yes. In any case, if we find someone that can test it and make a set of instructions we will add it to the wiki for sure!

**Comment by DarkHollow** [comment]
I also support this idea, I wish I had the knowledge to do it myself and contribute.

**Comment by Nemesis24** [comment]
Same for me i open a [discussion](https://github.com/community-scripts/ProxmoxVE/discussions/1927) about this.
[Proxmox VE Helper-Scripts](https://community-scripts.github.io/ProxmoxVE)

**Comment by brandong84** [comment]
[(https://github.com/brandong84/ProxmoxVE/blob/romm-install-script/install/romm-install.sh)](url)

I've modified a few of @gantoine scripts, and it now executes completely.  The romm service fails to start, I'm not familiar enough with how romm works to do much more troubleshooting by myself.  Here is the command to install it for anyone who has a Proxmox system that wants to try it out, and give any feedback or guidance.

**bash -c "$(wget -qLO - https://raw.githubusercontent.com/brandong84/ProxmoxVE/romm-install-script/ct/romm.sh)"**

**Reply by Moonraka** [reply]
I'll try this here in a bit.

**Reply by paraflu** [reply]
I'll give  a try too!

**Reply by godoffixxo** [reply]
Hey, I tryed it on pve 8 and it gives me following error:
<pre>
Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent

  💡  FIGlet or the slant font is missing. Installing...
  💡  Installing FIGlet...
xz: (stdin): File format not recognized
tar: Child returned status 1
tar: Error is not recoverable: exiting now

[ERROR] in line 199: exit code 0: while executing command tar -xf "$temp_dir/figlet.tar.xz" -C /tmp/figlet --strip-components=1

root@Zeus:~# 

</pre>
Thanks! :)

**Reply by SwedishLesbian** [reply]
Im getting the same error as godoffixxo

**Comment by frozen-cdn** [comment]
Finding the "quick start guide" to be anything but quick, and is actually rather annoying

I hate Docker, I don't use Docker, I don't care about docker.

Where are the instructions for Debian and Ubuntu?

Should be as simple as adding a repository and completing the relevent apt install command, and the convoluted password and hex nonsense should be automated, total UX failure

**Reply by Moonraka** [reply]
Bummer sounds like your outta luck with that attitude. If you change your mind come in discord and get support. Docker is what the development uses for their system. You could try apps inside Unraid, TrueNAS or other OS w/ App Stores of popular apps.

**Reply by gantoine** [reply]
<img width="593" alt="Screenshot 2025-06-05 at 11 19 34 PM" src="https://github.com/user-attachments/assets/56895a11-9ef6-4cbe-9fd3-281d667ef1aa" />

**Reply by zurdi15** [reply]
The fact that you don't seem to understand the majority of things and why are done that way (even if it can be improved of course) makes your comment absolutely irrelevant
